firmware/benchmark/csr_write/main.rs

```diff
@@ -0,0 +1,21 @@
+#![no_std]
+#![no_main]
+
+use core::arch::asm;
+
+use config_helpers::parse_usize_or;
+use miralis_abi::{setup_firmware, success};
+
+setup_firmware!(main);
+
+fn main() -> ! {
+    let iterations = parse_usize_or(option_env!("MIRALIS_BENCHMARK_NB_ITER"), 1);
```

Comment:
We will need to read this environment variable from multiple firmware images in the future, so we should probably factor it out in a single place. The `abi` crate is made for that, it defines functions for the firmware to interact with Miralis and other common utilities (such as initializing the stack, etc...)